storageClient/api.go

```diff
@@ -31,9 +31,10 @@ type Note struct {
 
 // CreateGroupRequest wraps parameters needed to request creation of a group
 type CreateGroupRequest struct {
-	Name              string `json:"group_name"`
-	PublicKey         string `json:"public_key"`
-	EncryptedGroupKey string `json:"encrypted_group_key"`
+	Name                   string `json:"group_name"`
+	PublicKey              string `json:"public_key"`
+	EncryptedGroupKey      string `json:"encrypted_group_key"`
+	EncryptedMembershipKey string `json:"encrypted_membership_key"`
```

Comment:
I thought the group and the membership key are the same thing? If not and we need this reminder to update the api-spec accordingly.

Reply:
You are correct! I got in the mindset of "must add membership key to this create group"